### PR TITLE
feat - Toast Documentation

### PR DESCRIPTION
**Pull request by Ayesha-7**
- added toast and its docs

### CODE REVIEW

File: Styles/Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  width: 50%;
+}
+.toast {
+  border: transparent 1px solid;
+  border-radius: 8px;
+  width: 70%;
+  padding: var(--sp-075);
+  font-size: 1.2rem;
+  background-color: var(--bg-primary);
+  box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.4);
```
Comment (SanjayJatti) [inline Styles/Components/toast.css:11]: You can create variable for box-shadow to be more consistent.

File: Styles/Components/toast.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.toast-container {
+  width: 50%;
+}
+.toast {
+  border: transparent 1px solid;
+  border-radius: 8px;
```
Comment (SanjayJatti) [inline Styles/Components/toast.css:6]: It is good that you didnot convert 'px' to 'rem' for small value as it doesn't make much difference.

**Review by SanjayJatti — APPROVED** [review]
All looking good

File: Styles/Components/toast.css (hunk 1 above)
Comment (Ayesha-7) [reply Styles/Components/toast.css:11]: thanks for the suggestion Sanjay!

**Pull request merged**